doc/classes/Quaternion.xml

```diff
@@ -164,7 +166,7 @@
 			<param index="1" name="weight" type="float" />
 			<description>
 				Returns the result of the spherical linear interpolation between this quaternion and [param to] by amount [param weight].
-				[b]Note:[/b] Both quaternions must be normalized.
+				[b]Note:[/b] Both quaternions must be normalized. Returns [code](0, 0, 0, 1)[/code] if any quaternion is not normalized in debug mode. Returns [code]self[/code] if any quaternion is not normalized in release mode.
```

Comment:
`self` is a GDScript-exclusive context and it only applies to Object, because all other Variant types cannot be extended.

In this situation the class reference likes to say, for example "Returns this quaternion unchanged" or "Returns the quaternion without changes" or similar.